src/Confluent.Kafka/Impl/NativeMethods/NativeMethods.cs

```diff
@@ -287,8 +290,7 @@ internal class NativeMethods
 
         [DllImport(DllName, CallingConvention = CallingConvention.Cdecl)]
         internal static extern SafeTopicHandle rd_kafka_topic_new(
-                IntPtr rk,
-                [MarshalAs(UnmanagedType.LPStr)] string topic,
+                IntPtr rk, IntPtr topic,
```

Comment:
since Kafka topics are constrained to be a subset of the ANSI characters, marshalling as LPStr is not deficient, but the capability to marshal UTF8 strings I added this PR, although not actually needed, is likely come in useful later, so we might as well leave this change in here.